ui/app/adapters/pki/crl.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { encodePath } from 'vault/utils/path-encoding-helpers';
+import ApplicationAdapter from '../application';
+
+export default class PkiCrlAdapter extends ApplicationAdapter {
+  namespace = 'v1';
+
+  _url(backend) {
+    return `${this.buildURL()}/${encodePath(backend)}/config/crl`;
+  }
+
+  findRecord(store, type, id) {
+    return this.ajax(this._url(id), 'GET').then((resp) => {
```

Comment:
would it be too much trouble to change the `id` param here to `backend` so it's more clear what it's expecting? ID could be any number of things, and it's usually not the backend path 😄